src/test/TaskListTest.java

```diff
@@ -0,0 +1,56 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Perhaps you could consider explicitly listing out the imported methods from the Assertions class. This would make it clearer.